doc/admin-guide/files/strategies.yaml.en.rst

```diff
@@ -182,9 +182,18 @@ Each **strategy** in the list may using the following parameters:
    #. **path**: (**default**) Creates a hash over the path portion of the request URL.
    #. **path+query**: Same as **path** but adds the **query string** in the request URL.
    #. **path+fragment**: Same as **path** but adds the fragment portion of the URL.
-   #. **cache_key**: Uses the hash key from the **cachekey** plugin.  defaults to **path** if the **cachekey** plugin is not configured on the **remap**.
+   #. **cache_key**: Deprecated, use the URL designation instead.
```

Comment:
Since we are deprecating/changing how some of this works it would be good if there was a deprecation note on what settings to use so that this can act how it used to pre-changes. Just helps with transition

Reply:
Added.